bin/kong

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env lua
+#!/usr/bin/env luajit
```

Comment:
iic the debian package to install kong includes LuaJit. So my guess is that it wouldn't break anything (both LuaRocks and busted are running against LuaJit). Or am I missing something?

Then again, I have no idea how the debian package is generated and whether other distro's behave differently.